### PR TITLE
[dhctl] add preflight check for python and his friends

### PR DESCRIPTION
**Pull request by mvasl**
## Description

Added a preflight check to validate that required python modules are installed on the node

## What is the expected result?

Installer will not begin bootstrapping nodes without required python modules

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: dhctl
type: feature
summary: dhctl will now check if required python modules are installed on the node before bootstrapping it
impact_level: default
```

### CODE REVIEW

**Comment by mvasl** [comment]
Снимок экрана 2024-07-02 в 14 10 48
Снимок экрана 2024-07-02 в 14 11 49

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/gcp'...

**Comment by mvasl** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.27 9760515594 9760515594-1-con-1-27 /sys/deckhouse-oss/install:pr8867

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by mvasl** [comment]
/e2e/abort azure;Standard;containerd;1.27 9760520525 9760520525-1-con-1-27 /sys/deckhouse-oss/install:pr8867

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by mvasl** [comment]
/e2e/abort static;Static;containerd;1.27 9760519197 9760519197-1-con-1-27 /sys/deckhouse-oss/install:pr8867 astra@192.168.199.244

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/aws'...

**Comment by mvasl** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.27 9761804748 9761804748-1-con-1-27 /sys/deckhouse-oss/install:pr8867 centos@3.73.68.51

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/azure'...

File: dhctl/pkg/preflight/python.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Flant JSC
```
Comment (name212) [inline dhctl/pkg/preflight/python.go:1]:
```suggestion
// Copyright 2024 Flant JSC
```

**Review by name212 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8889](https://github.com/deckhouse/deckhouse/pull/8889) to the branch [release-1.62](https://github.com/deckhouse/deckhouse/tree/release-1.62) successful!

**Pull request merged**